Style

zusoomro opened the pull request.

Commit:
Updates splash screen

Commit:
Syntactic changes, updates style

Commit:
Init work on pods

Commit:
Moves load pod into redux state

Commit:
Adds manage users page

Commit:
Adds image to schedule page

Commit:
Fixes bug which kept you signed in

In previous versions, a user might be shown the authenticated view (tab
bar, etc) without actually having a valid active user. This commit fixes
this bug and redirects the user to the login screen

Commit:
Removes podid from users

Commit:
Adds titles to pod screen names

Commit:
Move let to const for constant

allysmith12 commented on .gitignore:
```diff
@@ -0,0 +1,6 @@
+.DS_Store
```
Oh thank god.

allysmith12 commented on mobile/src/shared/Button.tsx:
```diff
@@ -0,0 +1,50 @@
+import {
```
Is this just a fancy shared button??

zusoomro replied:
Yea it's easier to style

allysmith12 reviewed (approved):
Sure, LGTM

Commit:
Merge branch 'view-members-of-pod' into resolve-pod-schedule

Commit:
Merge branch 'main' into resolve-pod-schedule

Commit:
Updates seed file to include events, adds script

Adds the "resetDb" script, which will reset the local dev database and
populate it with the database seed

Commit:
Updates pods to use redux actions

Commit:
Minor changes

- Stop autocaps in login, register
- Fix eslint again

Commit:
Adds types, fixes logout

- Fixes a bug in logout where token wasn't deleted
- Adds types for navigators

Commit:
Big changes towards pod resolution

- Changes api backend to remove hardcoding
- Adds events to seed
- Adds tests for conflictingEvents function
- Creates frontend for conflicting events

Commit:
Add types, minor work

Commit:
Merge branch 'resolve-pod-schedule' into style

Commit:
Adds avatar for current user, style changes, login refactor

- Changes login and register component to be the same for the time being

Commit:
Formats error message

Commit:
Modify/create event now share component

Commit:
Style pod creation flow, update redux state for pod creation

Commit:
Creates type for invite

Commit:
Adds email to pod events

Commit:
Refactors PodHomeScreen component

- Moves api calls which are not connected to the Redux state into
"podService"
- Moves modal component into PodInviteModal
- Fixes bug where accepting a pod invite didn't immediately put you in
the pod (the Redux action needed to be wrapped in a dispatch)

zusoomro commented:
This is the most recent version (that I demo'd) and has a bunch of changes. The branch name isn't representative of the work here. The changes can be viewed in the commit descriptions as well, but here's a high level list

- Design and style changes across the front end
- Use the same component for login and register for the time being
- Use the same component to create and modify an event
- Pods now have members, which are all the users in that pod

chloeprezelski commented:
Is there a reason this hasn't been merged yet?

allysmith12 commented:
Zulfi can we merge this so I can merge in my stuff please :)

Commit:
Merge branch 'main' into style-2

Commit:
Update style
